app/src/main/java/de/qspool/clementineremote/ui/fragments/LibraryFragment.java

```diff
@@ -89,6 +96,16 @@ public void onCreate(Bundle savedInstanceState) {
         setHasOptionsMenu(true);
 
         mUnknownItem = getString(R.string.library_unknown_item);
+        loadSettings();
```

Comment:
Move `loadSettings()` to `onResume()`. Otherwise the app might need a restart to apply the new config.